app_dart/lib/src/service/github_service.dart

```diff
@@ -159,6 +160,64 @@ class GithubService {
     );
   }
 
+  Future<List<PullRequestReview>> listPullRequestReviews(RepositorySlug slug, int prNumber) async {
+    return github.pullRequests.listReviews(slug, prNumber).toList();
+  }
+
+  /// Retrieves check runs with the ref.
+  Future<List<CheckRun>> getCheckRuns(
```

Comment:
I'm confused on this and the APIs added below - didn't we already have methods for doing this we were already using?

Reply:
Got it, that makes more sense. Thank you!